src/site/shared/circuitdesigner/impl/CircuitDesigner.ts

```diff
@@ -64,16 +64,12 @@ export class CircuitDesignerImpl<CircuitT extends Circuit> implements CircuitDes
         inputAdapter.subscribe((ev) => this.toolManager.onEvent(ev, this));
 
         // Attach tool renderers
-        let renderCleanup = () => {};
-        if (renderers) {
-            renderCleanup = this.circuit.addRenderCallback(({ renderer, options, circuit }) => {
-                renderers.forEach((toolRenderer) => {
-                    const curTool = this.toolManager.curTool;
-                    if (toolRenderer.isActive(curTool))
-                        toolRenderer.render({ renderer, options, circuit, curTool, input: inputAdapter.state });
-                });
-            });
-        }
+        const renderCleanup = this.circuit.addRenderCallback((renderArgs) =>
+            renderers.forEach((toolRenderer) => toolRenderer.render({
+                curTool: this.toolManager.curTool,
+                input:   inputAdapter.state,
+                ...renderArgs,
+             })));
```

Comment:
Can the last `));` be moved to the next line so it is a little clearer what they are closing?